src/components/20-molecules/dropdown/index.scss

```diff
@@ -102,16 +102,14 @@ axa-dropdown {
   /* FIXME: remove this selectivity hack once v1 CSS problem is systematically addressed, cf. https://github.com/axa-ch/patterns-library/issues/1343 */
   .m-dropdown__select-icon.m-dropdown__select-icon {
     display: block;
-    position: absolute;
-    top: 15px;
-    right: 15px;
+    position: absolute; // needed for native select: icon should be "behind" the select
+    top: ($dropdown-height / 2) - (24px / 2); // dropdown height / 2 = vertical centered; 24px = height of SVG; 24px / 2 = center of SVG should be center of dropdown
```

Comment:
could this be centered with semantically more appropriate mechanisms like flex-box?

Reply:
Tried that, but:
- its not that easy because of the native- and custom variant in one component with shared css classes.
- if we use flexbox and the user clicks on the native dropdown the blue isactive border is just around the label field. The icon on the right is missed out.

So i decided to make a small chance instead of refactoring the whole component.

Reply:
OK, accepted.